.prettierrc.cjs

```diff
@@ -1,30 +1,43 @@
-// Because both of these prettier plugins hijack the typescript parser,
-// we can only use one of them at a time!
-//
-// If we try to use both, the second will overwrite the first.
-//
-// What follows is a workaround stolen from:
-// https://github.com/tailwindlabs/prettier-plugin-tailwindcss/issues/31#issuecomment-1203264916
+/** @type {import('prettier').Config} */
+const pluginsConfig = {
+  plugins: [
+    require.resolve("@ianvs/prettier-plugin-sort-imports"),
+    require("prettier-plugin-tailwindcss"),
+  ],
+}
 
-const pluginImportSort = require("prettier-plugin-import-sort")
-const pluginTailwindcss = require("prettier-plugin-tailwindcss")
+/** @type {import('prettier').Config} */
+const importSortConfig = {
+  importOrder: [
+    // built-ins like `node:fs`
+    "<TYPES>^(node:)", // type imports
+    "<BUILT_IN_MODULES>", // imports
+    "", // a gap
 
-/** @type {import("prettier").Parser}  */
-const myParser = {
-  ...pluginImportSort.parsers.typescript,
-  parse:
-    pluginTailwindcss.parsers.typescript.parse,
-}
+    // anything which doesn't match any other rules
+    "<TYPES>", // type imports
+    "<THIRD_PARTY_MODULES>", // imports
+    "", // a gap
+
+    // local aliases / packages starting with one of these prefixes
+    "<TYPES>^(@common|@talisman|@ui|@tests|@extension/core|@extension/shared)(/.*)?$", // type imports
+    "^(@common|@talisman|@ui|@tests|@extension/core|@extension/shared)(/.*)?$", // imports
+    "", // a gap
+
+    // local `./blah/something` packages
+    "<TYPES>^[.]", // type imports
+    "^[.]", // imports
+  ],
```

Comment:
This plugin is actively maintained (unlike the previous one), and with this config it sorts the imports in a very similar way to our previous setup, with three changes:

- It separates `import type` statements above any `import` statements
- It uses a case-insensitive sort (so AbcDefG instead of ADGbcef)
- It lets us put our `@common`/`@ui`/etc aliases next to our `./relative` imports